app/src/main/java/org/mozilla/focus/exceptions/ExceptionMigrationMiddleware.kt

```diff
@@ -35,8 +37,10 @@ class ExceptionMigrationMiddleware(
     ) {
         next(action)
 
-        if (action is TabListAction) {
+        if (action is TabListAction || action is ToggleExclusionListAction) {
```

Comment:
Why is the toggle action needed here?

Reply:
Without migrating the changes the toggle stays off. Maybe there is a deeper bug in `RemoveExceptionUseCase`?

Reply:
Yeah, it should only be needed to migrate once. After that Gecko and our saved list should stay in sync. (Except when the last tab was closed, then we need to re-migrate when the next GeckoSession comes up).